content/v2.0/reference/release-notes/influxdb-cloud.md

```diff
@@ -10,9 +10,20 @@ aliases:
   - /cloud/about/release-notes
```

Comment:
@gunnaraasen @bpalani, I added a statement to clarify that Cloud updates happen daily and release notes are a compilation of monthly updates.
I left the term "release notes" in case folks are familiar with the term and looking for "release notes" also, it keeps us consistent with our other "product updates" pages.

Reply:
@gunnaraasen, I updated the page title, and left the "monthly updates" grouped with release notes